wpilibNewCommands/src/main/native/include/frc2/command/button/Trigger.h

```diff
@@ -140,18 +149,20 @@ class Trigger {
   Trigger WhileFalse(Command* command);
 
   /**
-   * Starts the given command when the signal falls to `false` and cancels
-   * it when the signal rises. Moves command ownership to the button
-   * scheduler.
+   * Starts the given command when the condition changes to `false` and cancels
+   * it when the condition changes to `true`. Moves command ownership to the
+   * button scheduler.
+   *
+   * <p>Doesn't re-start the command in-between. If the command should restart,
```

Comment:
What does "in-between" mean in this context?

Reply:
If the command ends while the trigger is still `true`, the command isn't restarted. 
I agree this could be clarified.